src/workloads/query/TimeseriesBlockProcessing.yml

```diff
@@ -0,0 +1,356 @@
+          Operations:
+            - OperationMetricsName:
+              OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          $and:
+                            [
+                              {
+                                "time":
+                                  { $lt: { ^Date: "2022-01-02T00:00:00" } },
+                              },
+                              { "obj.measurement2": { $gte: 25 } },
+                            ],
+                        },
+                    },
+                    { $count: "count" },
+                  ]
+
+  # TODO: The $or $eq on obj.measurement1 gets converted to a $in so this query can't run in
+  # block processing. When it runs through SBE "normally", the filter gets created in a random order
+  # leading to consistently inconsistent query durations depending on which field gets evaluated
+  # first. Evaluating the filter on obj.measurement1 is fastest, while evaluating the filter on
+  # obj.measurement3 is slowest.
+  # - Name: MatchOnMuipleNestedFields
+  #   Type: CrudActor
+  #   Database: *database
+  #   Threads: *threads
+  #   Phases:
+  #     OnlyActiveInPhases:
+  #       Active: [9]
+  #       NopInPhasesUpTo: *maxPhases
+  #       PhaseConfig:
+  #         Repeat: *repeat
+  #         Database: *database
+  #         Collection: *collection
+  #         Operations:
+  #           - OperationMetricsName:
+  #             OperationName: aggregate
+  #             OperationCommand:
+  #               Pipeline:
+  #                 [
+  #                   {
+  #                     $match:
+  #                       {
+  #                         $and:
+  #                           [
+  #                             {
+  #                               $or:
+  #                                 [
+  #                                   { "obj.measurement1": 4 },
+  #                                   { "obj.measurement1": 9 },
+  #                                 ],
+  #                             },
+  #                             { "obj.measurement2": { $lte: 175 } },
+  #                             { "obj.measurement3": { $gt: 100, $lt: 800 } },
+  #                           ],
+  #                       },
+  #                   },
+  #                   { $count: "count" },
+  #                 ]
+
+  # The $match will be handled by block processing, but the pipeline from $group onwards will be
+  # handled by regular SBE execution.
+  - Name: MatchGroupSort
+    Type: CrudActor
+    Database: *database
+    Threads: *threads
+    Phases:
+      OnlyActiveInPhases:
+        Active: [10]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationMetricsName:
```

Comment:
(Drive-by review)

Are you intentionally leaving the `OperationMetricsName` fields empty?
I'm not sure of the intended behavior. If this is intentional, a comment would probably help.

Reply:
There wasn't any particular reason for this, think I just copied this pattern from an existing workload. Is there a particular convention for what `OperationMetricsName` should be?